sensu-plugins-gpg.gemspec

```diff
@@ -28,15 +28,17 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsGpg::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin', '~> 2.0'
```

Comment:
this is a breaking change and should be called out. It should have a link to the `sensu-plugin` changelog like this: https://github.com/sensu-plugins/sensu-plugins-mongodb/pull/50/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR9

Reply:
I have added the breaking change to `CHANGELOG.md`.